src/Microsoft.Azure.WebJobs.Host/Executors/DefaultStorageAccountProvider.cs

```diff
@@ -187,6 +188,11 @@ private IStorageAccount StorageAccount
             return account;
         }
 
+        public Task<IStorageAccount> TryGetAccountAsync(string connectionStringName, CancellationToken cancellationToken)
```

Comment:
I split them because the name wasn't very informative as to what it was doing, and it seemed to better fit with separation of concerns